modules/conversion-center/module.php

```diff
@@ -144,12 +170,59 @@ public function __construct() {
 		}, 100 );
 	}
 
+	public function handle_click_tracking() {
+		$data = filter_input_array( INPUT_POST, [
```

Comment:
Are there maybe not built-in WordPress methods for all these `filter_input`?

Reply:
@louiswol94 normally I use filter_input to filter and sanitize access to the super globals, I am used to it as it is a requirement for WordPress VIP coding standards. I could access and sanitize variables globally if you prefer.

Reply:
@nicoladj77 Okay cool. We also have a method `_unstable_get_super_global_value()`, but I am fine with this 👌